$NetBSD: patch-config.h.in,v 1.1 2019/09/16 17:22:24 nia Exp $

Linux (with glibc at least) needs strings.h for strcasecmp and
_GNU_SOURCE for strdup.

--- config.h.in.orig	2018-07-30 11:06:03.000000000 +0000
+++ config.h.in
@@ -41,11 +41,14 @@
 // Define on SunOS and Solaris
 #undef SUN
 
+#define _GNU_SOURCE
+
 #include <sys/types.h>
 #include <stdbool.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
+#include <strings.h>
 #include <limits.h>
 #include <assert.h>
 #include <stdio.h>
